autobahn/wamp_invocation.hpp

```diff
@@ -195,13 +200,13 @@ class wamp_invocation_impl
      * Reply to the invocation with positional arguments.
      */
     template <typename List>
-    void result(const List& arguments);
+    void result(const List& arguments, bool intermediate = false);
```

Comment:
You're not using the new parameter in any examples in this commit, it's undocumented and I'm not clear what it's purpose is. I'd be hesitant to merge more complicated API without having a clear view of what benefits it brings, and that those benefits outweigh the complexity and loss of readability that an additional boolean brings here. Please explain.